src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -398,4 +398,29 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="Error_NoDgSpec" xml:space="preserve">
     <value>None or invalid DgSpec was passed to NuGet add package command.</value>
   </data>
+  <data name="Error_AddPkgErrorStringForImportedEdit" xml:space="preserve">
+    <value>Item : {0} for {1} in Imported file : {2}</value>
```

Comment:
Is this colon delimited format used anywhere else? I would expect this to feel very similar to the add project command if possible.